### PR TITLE
Add third party notice in packages

### PR DESCRIPTION
**Pull request by cdmihai**
The packages msbuild ships under should contain the third party notice.

### CODE REVIEW

File: build/NuGetPackages/Microsoft.Build.Runtime.nuspec [hunk 1]
```diff
@@ -33,61 +33,62 @@
...
+    <file src="$outputPath$Microsoft.WinFx.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.WorkflowBuildExtensions.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Microsoft.Xaml.targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Workflow.Targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
+    <file src="$outputPath$Workflow.VisualBasic.Targets" target="contentFiles\any\net46\" buildAction="None" copyToOutput="true"/>
 
     <!--
       contentFiles\any\netcoreapp1.0
     -->
-    <file src="$outputPathNetCore$MSBuild.dll" target="contentFiles\any\netcoreapp1.0\" buildAction="None" copyToOutput="true"/>/>
```
Comment (rainersigwald) [inline build/NuGetPackages/Microsoft.Build.Runtime.nuspec]: How was this working before?

Comment (AndyGerlicher) [inline build/NuGetPackages/Microsoft.Build.Runtime.nuspec]: I thought the same thing. Turns out by default it gets escaped and is completely ignored!

```C#
var doc = new XmlDocument();
doc.Load(new StringReader(@"<files> <file /> /> </files>"));
void print(XmlLinkedNode element) => Console.WriteLine($"{element.NodeType}: {element.Name} = {element.Value}");
doc.ChildNodes.Cast<XmlLinkedNode>().Traverse(element => element.ChildNodes.Cast<XmlLinkedNode>()).ForEach(print);
```
=>
```
Element: files =
Element: file =
Text: #text =  />
```
You get a nice `XmlText` element mixed in there and if you don't check for it you'll never know was there...

**Comment by cdmihai** [comment]
@dotnet-bot test Windows_NT Build for CoreCLR please

**Comment by cdmihai** [comment]
@dotnet-bot test OSX Build for CoreCLR please

**Pull request merged**